sdk/freertos_app_cpu0/src/main.c

```diff
@@ -410,3 +191,39 @@ void vApplicationGetTimerTaskMemory(StaticTask_t **ppxTimerTaskTCBBuffer,
      * configMINIMAL_STACK_SIZE is specified in words, not bytes. */
     *pulTimerTaskStackSize = configTIMER_TASK_STACK_DEPTH;
 }
+
+/*******************************************************************************/
+/*              S E T U P   I N T E R R U P T   S Y S T E M                    */
+/*******************************************************************************/
+int SetupInterruptSystem(XScuGic *IntcInstancePtr) {
+	int Result;
+
+	XScuGic_Config *IntcConfig;
+
+	// Initialize the interrupt controller driver so that it is ready to use.
+
+	IntcConfig = XScuGic_LookupConfig(INTC_DEVICE_ID);//ELS -- this is misleading! there is only one XScuGic device (0)
+	if (NULL == IntcConfig) {
+		return XST_FAILURE;
+	}
```

Comment:
What is the purpose of adding this here? This is done in `intr_init()` in `shared/sys/intr.c`